scripts/populate.mjs

```diff
@@ -460,11 +460,23 @@ const storeAnnouncement = async (content, accountId, signer) => {
 
   // create and store announcement of content to batch
   const announcement = core.announcements.sign(
-    core.announcements.createBroadcast(
-      accountId,
-      `${process.env.REACT_APP_UPLOAD_HOST}/${hash}.json`,
-      hash
-    ),
+    type === core.announcements.AnnouncementType.Broadcast
```

Comment:
this would be nice if we wrapped this its own function

Reply:
good call. done.